config/settings.yaml.example

```diff
@@ -66,6 +66,23 @@
 #  :dynflow:
 #    :enabled: true
 
+:telemetry:
+  # one of 'prometheus', 'statsd', 'rails' or 'off'.
```

Comment:
The prometheus provider should have a big warning there that it's not meant for production setup, to avoid some confusion and false expectations this could bring to the users.

Reply:
Done.